README.md

```diff
@@ -1,67 +1,105 @@
-# Why forking Mozilla go-cose?
-
-This fork of [Mozilla go-cose](https://github.com/mozilla-services/go-cose) adds [Sign1](https://tools.ietf.org/html/rfc8152#section-4.2) capabilities to the parent package.  We hope we can reconcile our fork quite quickly.  In the meantime, you can inspect the delta [here](https://github.com/mozilla-services/go-cose/compare/master...veraison:master).
-
 # go-cose
 
-[![GitHub CI](https://github.com/veraison/go-cose/workflows/ci/badge.svg)](https://github.com/veraison/go-cose/actions?query=workflow%3Aci)
-
-[![Coverage Status](https://github.com/veraison/go-cose/workflows/cover%20%E2%89%A589%25/badge.svg)](https://github.com/veraison/go-cose/actions?query=workflow%3A%22cover%20%E2%89%A589%25%22)
+[![go.dev](https://pkg.go.dev/badge/github.com/veraison/go-cose.svg)](https://pkg.go.dev/github.com/veraison/go-cose)
+[![tests](https://github.com/veraison/go-cose/workflows/ci/badge.svg)](https://github.com/veraison/go-cose/actions?query=workflow%3Aci)
+[![coverage](https://github.com/veraison/go-cose/workflows/cover%20%E2%89%A589%25/badge.svg)](https://github.com/veraison/go-cose/actions?query=workflow%3A%22cover%20%E2%89%A589%25%22)
```

Comment:
These two badges are confusing since they does not only reflect the status of the `main` branch but also all the pull requests. We might want to remove them.

Reply:
For what I've seen in the wild, this is standard practice for GH Actions badges, although not ideal.

Reply:
A good practice is to split the GH actions for main branch and pull request.

An example can be https://github.com/fxamacker/cbor

Reply:
Badges only show the status of the default branch, so bad PRs won't affect the badges in the README and won't confuse casual readers.

This fact is explained in the GH Actions docs: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge.

For example, last three https://github.com/google/go-github GH Actions jobs failed, but as they are on PR branch the badge is still green.